client/signup/config/steps.js

```diff
@@ -109,7 +109,7 @@ export default {
 
 	about: {
 		stepName: 'about',
-		providesDependencies: [ 'themeSlugWithRepo', 'siteTitle' ],
+		providesDependencies: [ 'designType', 'themeSlugWithRepo', 'siteTitle', 'surveyQuestion' ],
```

Comment:
I think you might also need to add designType to the call to SignupActions.submitSignupStep in the about step component.

It's pretty confusing that we pass designType around in two different ways -  something we should cleanup later across all flows I think.